public/app/features/manage-dashboards/components/PublicDashboardListTable/PublicDashboardListTable.tsx

```diff
@@ -41,17 +43,30 @@ export const PublicDashboardListTable = () => {
             </tr>
           </thead>
           <tbody>
-            {publicDashboards?.map((pd) => (
+            {publicDashboards?.map((pd: ListPublicDashboardResponse) => (
               <tr key={pd.uid}>
                 <td className={styles.titleTd}>
-                  <Tooltip content={pd.title} placement="top">
-                    <Link className={styles.link} href={`/d/${pd.dashboardUid}`}>
-                      {pd.title}
-                    </Link>
+                  <Tooltip
+                    content={!!pd.dashboardUid ? pd.title : 'Related dashboard was already deleted'}
```

Comment:
Nit- We're using !!pd.dashboardUid a lot. Maybe it's worth making a little helper named "isOrphaned" or something to clarify expectations about why this would be empty for future maintainers

Reply:
yeah, I thought it. Gonna change it